src/acl/mock.ts

```diff
@@ -104,7 +104,7 @@ export function addMockFallbackAclTo<T extends WithServerResourceInfo>(
   return resourceWithFallbackAcl;
 }
 
-function setMockAclUrl<T extends WithServerResourceInfo>(
+export function setMockAclUrl<T extends WithServerResourceInfo>(
```

Comment:
By exporting this function is becomes available to users of the library - I'm not sure if that's what we want to do? If not, it should be moved to `mock.internal.ts` (and prefixed with `internal_`).

Reply:
Right, I have to be more careful about this. And correct me if I'm wrong, but once (/if eventually) eveybody switches to ESM, this will no longer be an issue, because all the imports go through the import map, right ?

Reply:
889bdc0d5de67aa02fe3729ecda64032076d50a3

Reply:
Yes and no. Yes, we should no longer need the `internal_` prefix because developers can no longer just import functions from files not added to the exports map, but no, because this specific file _is_ added to our exports map, so any export in there will still be public.